docs/html/examples.html

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Examples</title>
+  <meta name="description" content="SmallDragon Composite Data Types">
+  <meta name="author" content="SmallDragon">
+
+  <link rel="icon" href="../img/favicon.ico">
+  
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+  <link rel="stylesheet" href="../css/style.css">
+</head>
+
+<body>
+
+<div class="container">
+	
+	<div class="row nav-bg">
+		<div class="col"><a href="../index.html">Home</a></div>
+	</div>
+	
+	<h1>Examples</h1>
+
+	<p>
+	Do you want some inspiration? Check out the following examples to get a first impression on how code looks 
+	like when using smalldragon.
+	</p>
+
+	<h3> Example 1: Harshad Numbers</h3>
```

Comment:
Harshad numbers were the perfect example for a beginner since this kata is quite easy and not too long.